test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -81,6 +82,16 @@ def test_pytorch_train_mnist_gpu(pytorch_training, ec2_connection, gpu_only, ec2
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_MNIST_CMD)
 
 
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.model("bert")
+@pytest.mark.integration("inductor")
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_GPU_INSTANCE_TYPE, indirect=True)
```

Comment:
Improve coverage to p3, p4, g5 and g4dn. Currently, Triton is not supported on G3 instances.